modules/local/pixelator/single-cell-pna/layout/main.nf

```diff
@@ -5,8 +5,8 @@ process PIXELATOR_PNA_LAYOUT {
     // TODO: Add conda
     // conda "bioconda::pixelator=0.18.2"
     container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
-        ? 'quay.io/pixelgen-technologies/pixelator:0.21.4'
-        : 'quay.io/pixelgen-technologies/pixelator:0.21.4'}"
+        ? 'quay.io/pixelgen-technologies/pixelator:0.22.1'
```

Comment:
Not a blocking request for this release, but for consistency,, but for consistency it would be good to have an nf.test script for this module as well

Reply:
Relatable :P: Good luck